README.md

```diff
@@ -1,2 +1,53 @@
-# Capstone-Module-2
-Web application based on an external API. The API provides data about a topic.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone Module 2
+
+
+> Web application that get data from https://www.themealdb.com/api.php maintains likes, comments and reservations using Involvement API.
+
+
+## Built With
+
+- Technologies used: HTML, CSS, Javascript, Webpack, Jest
+
+## Live Demo (if available)
+
+[Live Demo Link](https://neckerfree.github.io/Capstone-Module-2/)
+
+[Presentation Video](https://www.loom.com/share/51f6bd1fdc254014817f131c54404393)
```

Comment:
- [ ] **Required:** Presentation should be done by all members of the team. If there is a reason why you have done it alone. Kindly share it with the next reviewer.
- [ ] **Optional:** State an interesting part of this project that you implemented.

Reply:
I had some difficulties with my partner, and I was unpaired: That’s why I’m presenting this project individually.
I have updated my video.

Reply:
Awesome!

Reply:
This clear 👍🏽